index.html

```diff
@@ -37,13 +37,19 @@
             <div class="row">
                 <div class="input-field col s12 l6">
                     <i class="material-icons prefix">account_circle</i>
-                    <input id="icon_prefix" type="text" class="validate" name="firstName" required>
+                    <input id="icon_prefix" type="text" class="validate" style="text-transform: uppercase"
```

Comment:
All styles should be moved the CSS file, it is the law of separation of concerns.

Reply:
Styles should still be moved to stylesheets